pkg/controller/populators/populator-base.go

```diff
@@ -241,11 +246,26 @@ func (r *ReconcilerBase) updatePVCWithPVCPrimeAnnotations(pvc, pvcPrime *corev1.
 	if !reflect.DeepEqual(pvc.ObjectMeta, pvcCopy.ObjectMeta) {
 		err := r.client.Update(context.TODO(), pvcCopy)
 		if err != nil {
-			return err
+			return nil, err
 		}
 	}
 
-	return nil
+	return pvcCopy, nil
+}
+
+func (r *ReconcilerBase) updatePVCWithPVCPrimeLabels(pvc, pvcPrime *corev1.PersistentVolumeClaim) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
Does it have to return PVC? The pvc is not used in this commit (only err is used)

Reply:
Not strictly, but I stuck to the coding style of `updatePVCWithPVCPrimeAnnotation` above.